src/FieldWarning/Assets/Camera/MiniMap.cs

```diff
@@ -50,7 +50,9 @@ private void Start()
             transform.parent.parent.GetComponent<RectTransform>().rect.width;
 
         TerrainMap map = MatchSession.Current.TerrainMap;
-
+        if (map == null)
+            return;
```

Comment:
For these random null checks we should write a comment on why we added them. Something like

> // When loading a map, the camera is moved to the loading scene 
> // where no terrain is present. Don't throw errors in that case.
> if (map == null)
>     return;

Otherwise this code is very ??? when you encounter it.